x/mongo/driver/operation.go

```diff
@@ -309,6 +310,13 @@ func (op Operation) Validate() error {
 	return nil
 }
 
+var writeBuffers = sync.Pool{
```

Comment:
Since this pool is used for both reads and writes, consider renaming to `memoryPool`.